google/ggrc-core
# GGRC-6674 Disable possibility to add/edit custom roles in administration

# Issue description

Disable adding/removing/updating global custom roles for controls on administration dashboard.

# Steps to test the changes

1. Open 'Custom Roles' tab in 'Admin Dashboard'
2. Expand 'Contols'
3. Make sure icon to add custom roles is removed
4. Make sure 'Edit' item is removed for non-predefined roles

# Solution description

Control's model flag was introduced previously. Based on this flag functionality to add/update custom roles is disabled.

# Sanity checklist




# PR Review checklist
